.github/workflows/release.yml

```diff
@@ -17,6 +17,9 @@ jobs:
     name: Release
     runs-on: ubuntu-latest
     if: ${{ github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/heads/release/') }}
+    needs: 
```

Comment:
Does this work? Can you do this across workflows?

Reply:
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds